eng/common/pipelines/templates/jobs/docindex.yml

```diff
@@ -1,7 +1,7 @@
 jobs:
```

Comment:
`eng/common` should not be changed.

Reply:
It is auto PR, so it changed all occurrences. I will merge tools PR first once we are confident on all languages.